docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -1000,38 +917,22 @@ desired={<Thirteen viewBox="140 40 680 500" />}
 
 #### Current state
 
-Camunda 8 is pointing at the Elasticsearch instances in both regions again and not the temporary instance. It still remains unreachable to the end-users and no processes are advanced.
+Camunda 8 is reachable to the end-user but not yet exporting any data.
 
 #### Desired state
 
-You are reactivating the exporters and enabling Operate and Tasklist again within the two regions. This will allow users to interact with Camunda 8 again.
+You are reactivating the existing exporter and initializing a new exporter to the recreated region. This will allow Zeebe to export data to Elasticsearch again.
 
 #### How to get there
 
-1. Upgrade the normal Camunda environment in `CAMUNDA_NAMESPACE_SURVIVING` and `REGION_SURVIVING` to deploy Operate and Tasklist:
-
-```bash
-helm upgrade $HELM_RELEASE_NAME camunda/camunda-platform \
-  --version $HELM_CHART_VERSION \
-  --kube-context $CLUSTER_SURVIVING \
-  --namespace $CAMUNDA_NAMESPACE_SURVIVING \
-  -f camunda-values.yml \
-  -f $REGION_SURVIVING/camunda-values.yml
-```
-
-2. Upgrade the new region environment in `CAMUNDA_NAMESPACE_RECREATED` and `REGION_RECREATED` to deploy Operate and Tasklist:
+1. Initialize the new exporter for the recreated region by sending an API request via the Zeebe Gateway:
 
 ```bash
-helm upgrade $HELM_RELEASE_NAME camunda/camunda-platform \
-  --version $HELM_CHART_VERSION \
-  --kube-context $CLUSTER_RECREATED \
-  --namespace $CAMUNDA_NAMESPACE_RECREATED \
-  -f camunda-values.yml \
-  -f $REGION_RECREATED/camunda-values.yml \
-  --set global.multiregion.installationType=failBack
+kubectl --context $CLUSTER_SURVIVING port-forward services/$HELM_RELEASE_NAME-zeebe-gateway 9600:9600 -n $CAMUNDA_NAMESPACE_SURVIVING
+curl -XPOST 'http://localhost:9600/actuator/exporters/elasticsearchregion1/enable' -H 'Content-Type: application/json' -d '{"initializeFrom" : "elasticsearchregion0"}'
 ```
 
-3. Reactivate the exporters by sending the API activation request via the Zeebe Gateway:
+2. Reactivate the exporters by sending the API activation request via the Zeebe Gateway:
```

Comment:
It would be better to split this into a new step. We should resume only after the exporter has enabled. Enabling exporter usually finishes faster, but it is not a synchronous operation. So the process should be as follows.

1. enable exporter
2. verify it is enabled
3. Resume exporting